src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -344,7 +346,7 @@ public void onWorldUnload(World w)
      */
     public void onServerTick(TickEvent.ServerTickEvent event)
     {
-        for (Building b : buildings.values())
+        for (AbstractBuilding b : buildings.values())
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'b' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)